docs/docs/channels/chat/slack.md

```diff
@@ -6,42 +6,68 @@ This guide will walk you through the steps needed to obtain the `webhookUrl` tha
+2. Add the `Add to Slack` button or the shareable URL to your application in order to request permission of access (scope: incoming-webhook).
+   <br/>
+   Make sure to use the generated sharable URL that is located under Slack form in the <a href="https://web.novu.co/integrations" style={{textDecoration: "underline"}}>Integration Store</a>.
+
+### Manual Manage
+
+<details>
+<summary>Instructions on configuring https server</summary>
+
+Create a new endpoint on your server that will handle the following steps (you could you can use Request Bin for an easy HTTPS service for redirects):
+
+1. Listen for redirect requests to your endpoint (REDIRECT_URL) after the user completes step 5 and grants permissions. Make sure to store the 'code' parameter from the request query as it will be needed later.
+2. Send a POST request to <https://slack.com/api/oauth.v2.access> with the following request body:
+   Use the "Client ID" and "Client Secret" from Slack's Developer Dashboard under "Basic Information". The request body should be in the format: { code: string, client_id: string, client_secret: string }.
+   Store the webhook URL from the response, which can be found under response.data.incoming_webhook.url.
+   (read more on Slack's documentation here)
+3. When the incoming_webhook.url is obtained you need to save it on the relevant subscriber entity in Novu you can use the node sdk:
+
+   ```typescript
+   import { Novu, ChatProviderIdEnum } from '@novu/node';
 
-1. On your server add a new endpoint that will listen to the response redirect from a POST request to `https://slack.com/api/oauth.v2.access` (read more on Slack's documentation [here](https://api.slack.com/authentication/oauth-v2#asking))
-2. Go to `OAuth & Permissions` on Slack's developer dashboard and add your REDIRECT_URL in Redirect URLs. (You can use [Request Bin](https://requestbin.com/) for an easy HTTPS service for redirects)
-3. Go to `Manage Distribution` and at the bottom of the page make sure to tick `Remove Hard Coded Information` and `Activate Public Distribution`.
-4. Add the `Add to Slack` button or the shareable URL to your application.
-5. After the end-user finishes the authorization you can get the `webhookUrl` from the response of the OAuth under `body.incoming_webhook.url`.
-6. When the `incoming_webhook.url` is obtained we can save it on the relevant subscriber entity in Novu:
+   const novu = new Novu(process.env.NOVU_API_KEY);
 
-```typescript
-import { Novu, ChatProviderIdEnum } from '@novu/node';
+   const body = req.body; // From your HTTPS listener
+   await novu.subscribers.setCredentials('subscriberId', ChatProviderIdEnum.Slack, {
+     webhookUrl: body.incoming_webhook.url,
+   });
+   ```
 
-const novu = new Novu(process.env.NOVU_API_KEY);
+   - subscriberId is a custom identifier used when identifying your users within the Novu platform.
+   - providerId is a unique provider identifier. We recommend using our ChatProviderIdEnum to specify the provider.
```

Comment:
We have a lot of non-JS/TS users, can we replace ChatProviderEnum here with the actual values too?

So something like:

`providerId is a unique provider identifier. We recommend using our ChatProviderIdEnum (if you're using Node), or any of the following values (Slack, ..., ...) to specify the provider.`